spec/lib/contextual_logger/logger_with_context_spec.rb

```diff
@@ -99,6 +99,22 @@
       end
     end
 
+    context "when string passed as context key" do
+      it "returns context with a symbol key" do
```

Comment:
To keep tests as small and specific as possible it is best to split this out into two separate tests.